viewer/static_src/css/crawler.less

```diff
@@ -0,0 +1,65 @@
+.o-radio_buttons {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+
+  div {
+    flex: 50%;
+  }
+
+  label {
+    font-weight: bold;
+
+    > span {
+      display: block;
+      margin-top: 3px;
+      margin-bottom: 15px;
+      font-weight: normal;
+    }
+  }
+}
+
+.results_list {
+  li {
+    list-style-type: none;
+  }
+
+  .results_item {
+    padding: 24px 0 !important;
+
+    + .results_item {
+      border-top: 1px solid var(--gray-40);
+    }
+  }
+}
```

Comment:
The `o-radio_buttons` and `results_list` etc should also be updated to new BEM at some point. Maybe something to do in a future sass PR?

Reply:
That would look like `o-radio-buttons` (or maybe just `o-form__group` would be better?) and `results-list`, `results-list__item`?

Reply:
Thanks @anselmbradford, implemented in aa1d5dd!